### DISCUSSION TITLE
Improve  Longhorn performance on aws eks with IO2

### DISCUSSION THREAD

**Discussion by Elias19x** [longhorn/longhorn#3971] [Q&A]
Hello everyone  am testing longhorn perf in our cluster I got it running on 3 ec2 nodes C5a.4xlarge and each with 500gb of gp2 , first of all I got better performance results with gp2 than with io2 is  if there is any extra configuration to do to get the best out of longhorn that  you can please share ?

**Comment by PhanLe1010** [comment]
gp2 gets better result than io2? What are the steps that you have taken?

> extra configuration to do to get the best out of longhorn that you can please share

Beside of the disk, inter VM network is also important
